ui/components/MesherySnackbarWrapper.js

```diff
@@ -128,18 +131,26 @@ function MesherySnackbarWrapper(props) {
   const [socialExpand, setSocialExpand] = useState(false);
   const [anchorEl, setAnchorEl] = useState(null);
   const [socialMessage, setSocialMessage] = useState("");
+  const [highlight, setHighlight] = useState(false)
 
   const handleExpandClick = () => {
     setExpanded(!expanded);
+    // expand && (expand = false);
```

Comment:
lets delete the commented code if it not providing any help.

Reply:
Done 👍🏽